README.md

```diff
@@ -52,7 +52,7 @@ Many of these features require the [AWS SAM CLI](https://github.com/awslabs/aws-
 * **Run/Debug Local Lambda Functions** - Locally test and step-through debug functions in a Lambda-like execution environment provided by the [AWS SAM CLI](https://github.com/awslabs/aws-sam-cli)
 * **Invoke Remote Lambda Functions** - Invoke remote functions using a sharable run-configuration
 * **Package & Deploy Lambda Functions** - Ability to package a Lambda function zip and create a remote lambda
-* **Deploy SAM-based Applications** - (**Python only**) Package, deploy & track SAM-based applications
+* **Deploy SAM-based Applications** - (**Java & Python only**) Package, deploy & track SAM-based applications
```

Comment:
Can we move the **Java & Python only** to all the applied items, namely all the items except **Invoke Remote Lambda Functions**? Or simply remove this note and just mention it in the first paragraph, something like **SAM features only support Java & Python**?

Reply:
k